Sources/App/Models/Repository.swift

```diff
@@ -122,6 +119,9 @@ final class Repository: @unchecked Sendable, Model, Content {
 
     @Field(key: "summary")
     var summary: String?
+    
+    @Field(key: "forked_from")
+    var forkedFrom: Fork?
```

Comment:
Nit: we're _trying_ (😅) to keep the data fields sorted (although this isn't true generally, because it's something we started half-way through without going back and fixing it everywhere). I'll patch this up real quick.